compiler/rustc_codegen_llvm/src/lib.rs

```diff
@@ -74,6 +75,24 @@ mod value;
 
 rustc_fluent_macro::fluent_messages! { "../messages.ftl" }
 
+macro_rules! TryFromU32 {
+    derive() ($(#[$meta:meta])* $vis:vis enum $Type:ident {$(
+        $(#[$varmeta:meta])*
+        $Variant:ident $(= $discr:expr)?
+    ),*$(,)?}) => {
```

Comment:
Nit: I find this compact style quite hard to read.

```suggestion
    derive() (
        $(#[$meta:meta])*
        $vis:vis enum $Type:ident {
            $(
                $(#[$varmeta:meta])*
                $Variant:ident $(= $discr:expr)?
            ),* $(,)?
        }
    ) => {
```